ui/apps/platform/src/Containers/Vulnerabilities/hooks/useCanSnoozeCves.ts

```diff
@@ -0,0 +1,12 @@
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import usePermissions from 'hooks/usePermissions';
+
+export default function useCanSnoozeCves(): boolean {
```

Comment:
Observations, not necessarily a change request.
1. A hook minimizes the friction to add the niche feature to components. Bravo!
2. Because this pattern deserves imitation elsewhere, is there a naming convention that balances:
    * semantic meaning for conditional rendering: can snooze CVEs
        Even if the next item does not produce anything better, `LegacySnooze` would help me on this point.
    * awareness that it depends on permissions and feature flags
        One direction for brainstorming is prefix `can` versus `has` versus `is`

Reply:
How about `useHasLegacySnoozeCapability()`?

Reply:
Getting warmer. Is there a synonym for `Capability` which has related but different meaning:
https://github.com/stackrox/stackrox/blob/master/proto/api/v1/metadata_service.proto#L72-L102

Reply:
We could slim it down and remove the `Cap`: `useHasLegacySnoozeAbility`.

Or the more sophisticated sounding: `useHasLegacySnoozeCriterion`.

Reply:
Oh, yeah! Snooze Ability for the win.